modular_skyrat/modules/customization/modules/clothing/under/misc.dm

```diff
@@ -686,6 +686,30 @@
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
 	icon_state = "christmasfemaleg"
 
+/obj/item/clothing/under/rank/civilian/lawyer/beige/skirt/demon
+	name = "demonic suit"
+	desc = "Stylish enough to impress the devil."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
+	icon_state = "lucifer"
+	obj_flags = UNIQUE_RENAME
+	unique_reskin = list(
+		"Lucifer" = "lucifer",
+		"Justice" = "justice",
+		"Malina" = "malina",
+		"Pandemonica" = "pande",
+		"Modeus" = "modeus",
+		"Zdara" = "zdara",
```

Comment:
```suggestion
		"Envy" = "zdara",
```

Reply:
because we gotta follow "POLICY" or whatever


I personally still think all this is fine but
well shit